components/properties/src/bc.rs

```diff
@@ -0,0 +1,71 @@
+// This file is part of ICU4X. For terms of use, please see the file
```

Comment:
`nit`: please, name this file `src/bidi.rs` - `bc` becomes very cryptic for readers.

Reply:
done